tests/fn9.dora

```diff
@@ -0,0 +1,11 @@
+fun main() -> int {
+  return myadd(1, 2) - 3;
+}
+
+fun myzero() -> int = 0 * (42 - 23);
+
+fun myfour() -> int = return 4;
+
+fun myadd(x: int, y: int) -> int = x + y + myfour() * myzero();
+
+fun mythrow(x: int) -> int = throw "foo";
```

Comment:
Right now this is only checked at runtime, but a function that can throw an exception should be declared as `throws`. This might be a failure in the future.